src/aurelia-binding.d.ts

```diff
@@ -134,17 +134,17 @@ export declare interface CollectionObserver {
  */
 export declare enum bindingMode {
   /**
-   * Updates the binding target once. This is essentially a simpler form of one-way binding
+   * Updates the binding target once. This is essentially a simpler form of to-view binding
    * that provides better performance in cases where the source value does not change.
    */
   oneTime = 0,
   /**
    * Updates the binding target (target) property when the binding source (source) changes.
    * This type of binding is appropriate if the element being bound is implicitly read-only.
-   * If there is no need to monitor the changes of the target property, using the OneWay
+   * If there is no need to monitor the changes of the target property, using the to-view
    * binding mode avoids the overhead of the two-way binding mode.
    */
-  oneWay = 1,
+  toView = 1,
```

Comment:
The addition of `toView` here is good. We need to keep `oneWay` here for now, it's not fully deprecated.